src/main/java/AppConfig.java

```diff
@@ -0,0 +1,46 @@
+import bank.*;
+import bank.clerk.*;
+import person.Person;
+
+public class AppConfig {
+
+    private final CentralBank centralBank = new CentralBank();
+    private final BankStorage bankStorage = new BankStorage(centralBank);
+    private final BankSystem bankSystem = new BankSystem(bankStorage);
+
+    public Person person(){
+         return new Person(bankKiosk());
+    }
+    public BankKiosk bankKiosk(){
+        return new BankKiosk(bankClerkFacade());
+    }
+    public BankServiceMediator bankClerkFacade(){
+        return new BankServiceMediator(bankClerk(), depositClerk(), withdrawClerk(), remittanceClerk(), changeStatusClerk(), getAccountInfoClerk());
+    }
+
+    public CreateAccountClerk bankClerk() {
+        return new CreateAccountClerk(bankSystem);
+    }
+    public DepositClerk depositClerk() {
+        return new DepositClerk(bankSystem);
+    }
+    public WithdrawClerk withdrawClerk() {
+        return new WithdrawClerk(bankSystem);
+    }
+    public RemittanceClerk remittanceClerk() { return new RemittanceClerk(bankSystem);}
```

Comment:
```suggestion
    public RemittanceClerk remittanceClerk() { 
        return new RemittanceClerk(bankSystem);
    }
```
가독성을 높이기 쉽게 컨벤션을 신경 쓰는 것도 중요합니다